src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/prisoners/PrisonersResourceIntTest.kt

```diff
@@ -22,6 +24,122 @@ class PrisonersResourceIntTest : IntegrationTestBase() {
   @Autowired
   private lateinit var nomisDataBuilder: NomisDataBuilder
 
+  @AfterEach
+  fun cleanUp() {
+    repository.deleteOffenders()
+  }
+
+  @Test
+  fun `DSL setup check - latest booking on root`() {
+    lateinit var rootOffender: Offender
+    lateinit var aliasOffender1: Offender
+    lateinit var aliasOffender2: Offender
+    lateinit var activeBooking: OffenderBooking
+    lateinit var oldBooking: OffenderBooking
+    lateinit var aliasBooking: OffenderBooking
+    lateinit var oldAliasBooking: OffenderBooking
+
+    nomisDataBuilder.build {
+      rootOffender = offender(nomsId = "A1234TT") {
+        activeBooking = booking {}
+        oldBooking = booking(active = false)
+
+        aliasOffender1 = alias(lastName = "ALIAS1") {
+          aliasBooking = booking(active = false)
+          oldAliasBooking = booking(active = false)
```

Comment:
We still have a bug in the builders such that we can add aliases to aliases... e.g. we can do
```
aliasOffender1 = alias(lastName = "ALIAS1") {
  aliasOffenderNested1 = alias(lastName = "ALIASNESTED") {
    aliasOffenderNested2 = alias(lastName = "ALIASNESTEDNESTED") {
...
```
This is why we were advocating for adding an `AliasDSL` - this should be a copy of `OffenderDSL` but without the `alias` method (thus preventing nested aliases).

Reply:
I did wonder about that, ok ill make aliases a separate builder